lib/default_hostgroup_managed_host_patch.rb

```diff
@@ -50,11 +50,14 @@ def import_host_and_facts_with_match_hostgroup hostname, facts, certname = nil,
 
     def group_matches?(fact)
       fact.each do |fact_name, fact_regex|
-        fact_regex.gsub!(/(\A\/|\/\z)/, '')
+        fact_regexs = Array(fact_regex)
         host_fact_value = @host.facts_hash[fact_name]
-        Rails.logger.info "Fact = #{fact_name}"
-        Rails.logger.info "Regex = #{fact_regex}"
-        return true if Regexp.new(fact_regex).match(host_fact_value)
+        fact_regexs.each do |fact_regex|
```

Comment:
I still need to use Array(), otherwise I would have to call flatten since a nested array will occur whereas Array() takes care of this for me automatically.